test-apps/display-test-app/src/frontend/Notifications.ts

```diff
@@ -80,14 +80,14 @@ export class Notifications extends NotificationManager {
 
     const promise = new Promise<MessageBoxValue>((resolve, _rej) => {
       button.addEventListener("click", () => {
-        dialog.close();
+        (dialog as any).close(); // @WBG comment

```

Comment:
Does `@WBG comment` mean you intend to revisit this before completing the PR?

Reply:
🤦‍♂️ gah, yeah that's exactly what that means.  Thanks for catching it.

And yeah, seems they were removed due to a lack of browser support.  I figured the cast here was fine since it's just a test app?  Unless somebody wants to re-write this all to avoid using the dialog element...

Reply:
They both continue to work in chrome, firefox, and electron, so I'm fine with the cast.